testng-core/src/main/java/org/testng/internal/invokers/MethodInvocationHelper.java

```diff
@@ -417,6 +423,18 @@ private static void invokeWithTimeoutWithNewExecutor(
     }
   }
 
+  private static StackTraceElement[] getRunningMethodStackTrace(ExecutorService exec) {
+    if (!(exec instanceof ThreadPoolExecutor)) return null;
```

Comment:
Can we please wrap all these simple conditions within a `{}` for the sake of readability ?

Reply:
Autostyle moved the single condition block without `{}` on the same line, making it harder to read (see https://github.com/cbeust/testng/pull/2673/commits/e360c893e89cace9bdb1802080543038400fe152).
Ideally, this formatting issue should be handled by autostyle, but I will manually fix this.